lib/src/components/NxFilterInput/__tests__/NxFilterInput.test.tsx

```diff
@@ -19,44 +19,12 @@ describe('NxFilterInput', function() {
     expect(component).toHaveClassName('nx-filter-input');
   });
 
-  describe('when value prop is truthy', function() {
-    it('renders the clear icon in the add-on', function() {
-      const icon = shallowComponent({ value: 'test value' }).find(NxFontAwesomeIcon);
-      expect(icon).toExist();
-      expect(icon).toHaveProp('icon', faTimes);
-    });
-
-    it('calls onClear if the icon is clicked', function() {
-      const onClear = jest.fn(),
-          iconWrapper = shallowComponent({
-            value: 'test value',
-            onClear
-          }).find('.nx-filter-input__add-on');
-
-      expect(iconWrapper).toExist();
-      iconWrapper.simulate('click');
-      expect(onClear).toHaveBeenCalled();
-    });
-  });
-
   describe('when value prop is an empty string', function() {
```

Comment:
This `describe` wrapper should probably removed, since the `it` that it contains is always supposed to be true now

Reply:
> This `describe` wrapper should probably removed, since the `it` that it contains is always supposed to be true now

https://github.com/sonatype/sonatype-react-shared-components/pull/62/commits/ef38611f63239b49c2eed85e6985cf2941a3ab19